src/test/java/jraft/impl/RaftServerContextImplTest.java

```diff
@@ -1,18 +1,34 @@
 package jraft.impl;
 
 import jraft.RaftServerContext;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.*;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
 
+import java.rmi.server.ExportException;
 import java.util.LinkedList;
 import java.util.List;
+import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.TimeUnit;
 
 /**
  * Created by Chen on 7/31/17.
  */
+@RunWith(JUnit4.class)
 public class RaftServerContextImplTest {
+    static RaftServerContextImpl raftServerContext;
+    static final String serverName1 = "server1";
+    static final String serverName2 = "server2";
+    static final String serverIpAndPort = "localhost:50050";
+
+    @BeforeClass
```

Comment:
are you sure you want all tests share same server context? or you want @Before so each test, it would initialize again.